andela/blackninjas-frontend
# #169817815 Users should be able to log in using Facebook or Google

#### What does this PR do?
It implements the functionality of signing in with social accounts
#### Description of Task to be completed?
- add a button that calls the google consent screen
- add a Facebook button
#### How should this be manually tested?
- clone this repo on your local computer
- make sure you have all the packages installed by typing `npm install` in the terminal
- start the server by running `npm start` then browse to this route` localhost:8080/auth` and click on google and it should send the consent screen, choose an account and it will redirect you on the homepage
#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
[Starts #169817815](https://www.pivotaltracker.com/story/show/169817815)
#### Screenshots (if appropriate)
![ezgif com-optimize](https://user-images.githubusercontent.com/51836855/73931875-c5b47180-48e1-11ea-9fb3-22dc62310093.gif)

**When Redirected**
![image](https://user-images.githubusercontent.com/51836855/73931968-f7c5d380-48e1-11ea-94ab-9077c6194f1b.png)

#### Questions:
